### PR TITLE
feat(connect): ethereum definitions for getAddress

### LINKED ISSUE #8094: connect: ethereum definitions - getAddress
- implement ethereum definitions for ethereumGetPublicKey and ethereumGetAddress
 - add a test that requires eth definitions, see #8091

### PR DESCRIPTION
**Pull request by karliatto**
## Description

Ethereum definitions for ethereumGetAddress.

## Related Issue

Resolve https://github.com/trezor/trezor-suite/issues/8094

### CODE REVIEW

File: packages/transport/src/types/messages.ts [hunk 1]
```diff
@@ -554,8 +554,8 @@ export type FirmwareErase = {
 
 // FirmwareRequest
 export type FirmwareRequest = {
-    offset?: number;
-    length?: number;
+    offset: number;
```
Comment (mroz22) [inline packages/transport/src/types/messages.ts]: I'd suggest moving these changes to another PR. we shouldn't need to update submodules -> messages.json at all. do we?

File: packages/transport/scripts/protobuf-patches/index.js [hunk 2]
```diff
@@ -112,6 +112,7 @@ const TYPE_PATCH = {
     'Features.experimental_features': 'boolean | null',
     'HDNodePathType.node': 'HDNodeType | string',
     'FirmwareUpload.payload': 'Buffer | ArrayBuffer',
+    'EthereumGetAddress.encoded_network': 'ArrayBuffer',
```
Comment (mroz22) [inline packages/transport/scripts/protobuf-patches/index.js:115]: we may only need this part from all the protobuf related changes and also resulting changes from types

File: packages/transport/src/types/messages.ts (hunk 1 above)
Comment (karliatto) [reply packages/transport/src/types/messages.ts]: Not really, but I did not want to update it manually so I changed the patch for `EthereumGetAddress` to have the right type ` encoded_network?: ArrayBuffer;` and this is the result of the automatic script. 
But you are right, it is not part of this PR. So I am including only the required bit.

File: packages/transport/scripts/protobuf-patches/index.js (hunk 2 above)
Comment (karliatto) [reply packages/transport/scripts/protobuf-patches/index.js:115]: Yes, I updated, that is the way it is now 4c14e8fcae0f37c774f972e493e103350952880f

**Comment by mroz22** [comment]
I'd say that we are going to need to update also `GetAccountInfo` info for Ethereum so that it provides definitions for DeviceCommands.getAccountsDescriptor -> DeviceCommands.ethereumGetAddress. But it can be done as a follow.

**Review by mroz22 — APPROVED** [review]
seems ok, 

I just don't like passing path to path to `getEthereumDefinitions` but I understand why you do it. It would be nicer if it could get only slip44 segment. We can discuss this later. 

Also I would like to suggest getEthereumDefinitions to have only one argument object instead of couple arguments. 

Lets resolve those notes in a follwup

**Comment by karliatto** [comment]
> seems ok,
> 
> I just don't like passing path to path to `getEthereumDefinitions` but I understand why you do it. It would be nicer if it could get only slip44 segment. We can discuss this later.
> 
> Also I would like to suggest getEthereumDefinitions to have only one argument object instead of couple arguments.
> 
> Lets resolve those notes in a follwup

Yes, that is a good point. I created this PR https://github.com/trezor/trezor-suite/pull/8152 as a follow up. Is that what you meant?
